torch/testing/_internal/common_optimizers.py

```diff
@@ -599,6 +599,9 @@ def optim_error_inputs_func_nadam(device, dtype):
 
 # Weird story bro, NAdam and RAdam do not have maximize.
 def optim_inputs_func_radam(device=None):
+    cuda_supported_configs = [
+        OptimizerInput(params=None, kwargs={"capturable": True}, desc="capturable"),
+    ]
```

Comment:
It'd be good to add one for weight decay + another for weight decay + decoupled weight decay

Reply:
Oh does it matter that these are covered in the cudagraph tests?

Reply:
Discussed offline, I can add